modules/tensor_mechanics/src/materials/ADHillCreepStressUpdate.C

```diff
@@ -30,6 +30,10 @@ ADHillCreepStressUpdate::validParams()
   params.addRequiredParam<Real>("activation_energy", "Activation energy");
   params.addParam<Real>("gas_constant", 8.3143, "Universal gas constant");
   params.addParam<Real>("start_time", 0.0, "Start time (if not zero)");
+  params.addParam<bool>("use_transformation",
+                        false,
+                        "Whether to employ updated Hill's tensor due to rigid body or large "
+                        "deformation kinematic rotations");
```

Comment:
We probably would be able to use the optimized version for some transformation angles like 90, 180 etc. Maybe we should add a comment regarding that.